samples/subsys/mgmt/mcumgr/smp_svr/overlay-fs.conf

```diff
@@ -8,3 +8,5 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2304
 
 # Enable file system commands
 CONFIG_MCUMGR_GRP_FS=y
+
+CONFIG_REQUIRES_MALLOC=y
```

Comment:
Likewise with this, should go to the subsys Kconfig, not the same, this would be a breaking change to every out of tree user using any of these modules/drivers/subsystems because suddenly after updating their applications would entirely just break if it is left like this